### PR TITLE
Support latest chrome driver version

### PR DESCRIPTION
**Pull request by alexander-gridnev**
* addresses the following issue https://github.com/SergeyPirogov/webdriver_manager/issues/443.
* fixes Pipfile ("name" field became required for index definition)
* update tests to support execution on m1/m2 mac CPUs.

### CODE REVIEW

**Comment by matthiasha** [comment]
Thanks Alexander, this works for my colleague!

**Comment by rain01** [comment]
Thank you Alexander. Any idea when this will be released? Is there a workaround until then?
I tried using `latest_release_url` but it seems this is rather just used for version checking not passing url of the latest download link?

**Comment by Jfromtheroc** [comment]
Having the same issue, thank you

**Comment by duncanariey** [comment]
I'm also seeing this issue. Thank you for the quick fix, would love to see it released soon!

**Comment by alexander-gridnev** [comment]
@rain01 yeah, latest_release_url it's the url which used to check latest available version (returns string "106.0.5249.61" atm), so, i just use direct link to the commit in requirements files (like this - git+https://github.com/alexander-gridnev/webdriver_manager@62671021ee5456c315d9d0eae3f2428d6cec955f#egg=webdriver-manager)

Hey @aleksandr-kotlyar @SergeyPirogov, could you guys review the change?

File: webdriver_manager/drivers/chrome.py [hunk 1]
```diff
@@ -29,10 +30,22 @@ def get_os_type(self):
             return os_type
 
         if is_arch(os_type):
-            return f"{os_type.replace('_m1', '')}_m1"
+            return "mac_arm64"
```
Comment (SergeyPirogov) [inline webdriver_manager/drivers/chrome.py:33]: ok, but if it's arch that is not mac?

Comment (alexander-gridnev) [reply webdriver_manager/drivers/chrome.py:33]: previous if statement checks "none mac" case, and returns os_type as is.

But yeah, is_arch function is kind of misleading atm, i was even thinking to rename to "is_mac_arch" and add to inside function check if that's mac or not, cause it's always used after  is_mac check in current codebase

**Comment by SergeyPirogov** [comment]
Ok will merge it. @alexander-gridnev thank you

**Comment by jakemraz** [comment]
[https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip](https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip) has not been found.

but we can find  [ https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac_arm64.zip](https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac_arm64.zip)

after this patch was applied, my webdriver manager made a following error
"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip"

Please check this

**Comment by VladimirPodolyan** [comment]
@SergeyPirogov can you please release this fix in pypi as soon as possible ?

**Comment by ambitionist** [comment]
> https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip has not been found.
> 
> but we can find [ https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac_arm64.zip](https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac_arm64.zip)
> 
> after this patch was applied, my webdriver manager made a following error "ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip"
> 
> Please check this

Having the same problem here...

**Comment by Pr0s3lyt3** [comment]
Same issue as well, please assist!

**Comment by alexander-gridnev** [comment]
Hi @Pr0s3lyt3, @ambitionist, could show output of such snippet:

```
from webdriver_manager.chrome import ChromeDriverManager
from webdriver_manager.core.utils import ChromeType

manager = ChromeDriverManager(chrome_type=ChromeType.CHROMIUM)

print(manager.driver.get_os_type())
print(manager.driver.get_version())
print(manager.driver.get_url())
```

Also, on which macs do you run it?

**Comment by ambitionist** [comment]
> 
Output: 
mac64_m1
106.0.5249.61
https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip

I'm using a mac m1 2020 with Monterey 12.6

**Comment by alexander-gridnev** [comment]
hmm, could you show output `pip freeze | grep webdriver-manager` in shell? I feel like you still have wrong version

**Comment by alexander-gridnev** [comment]
here is what i see:

```
# pip freeze | grep webdriver-manager
webdriver-manager @ git+https://github.com/alexander-gridnev/webdriver_manager@62671021ee5456c315d9d0eae3f2428d6cec955f
```

**Comment by ambitionist** [comment]
> here is what i see:
> 
> ```
> # pip freeze | grep webdriver-manager
> webdriver-manager @ git+https://github.com/alexander-gridnev/webdriver_manager@62671021ee5456c315d9d0eae3f2428d6cec955f
> ```

```
# pip freeze | grep webdriver-manager                                                                           
webdriver-manager==3.8.3
```

**Comment by alexander-gridnev** [comment]
try to uninstall and install current master:

```
pip uninstall webdriver_manager
pip install git+https://github.com/SergeyPirogov/webdriver_manager@master
```

**Comment by ShasTheMass** [comment]
I confirm this is what I saw too 

```
$ pip freeze | grep webdriver
webdriver-manager==3.8.3
```

But if you do this instead:
```
$ pip install git+https://github.com/SergeyPirogov/webdriver_manager.git@master#egg=webdriver-manager
```
then you shall get:
```
$ pip freeze | grep webdriver
webdriver-manager @ git+https://github.com/SergeyPirogov/webdriver_manager.git@dd39e872f722b6f3e43d9146f25b1983231ccb80
```

**Comment by cmin764** [comment]
Can we please release this as asked [here](https://github.com/SergeyPirogov/webdriver_manager/issues/446#issuecomment-1274558712) as well?

File: webdriver_manager/drivers/chrome.py (hunk 1 above)
Comment (cmin764) [reply webdriver_manager/drivers/chrome.py:33]: Thanks for the fix!

I always wondered, why the function is named `is_arch` and it isn't `is_arm`? Isn't `is_arch` coming from is **arch**itecture (which can be any arch)?

**Comment by EwoutH** [comment]
I'm having this issue as well. @SergeyPirogov could you please tag a release?

**Comment by SergeyPirogov** [comment]
Hello, could you provide a pr with version update to 3.8.4

**Comment by EwoutH** [comment]
> Hello, could you provide a pr with version update to 3.8.4

That's normally something that a maintainer does.

But if you want that, is there a maintenance branch I can merge into?

**Comment by chrisgraham24** [comment]
I am still seeing this issue -- ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/106.0.5249.61/chromedriver_mac64_m1.zip

**Comment by mpineyrozaelot** [comment]
Hello guys, any idea on when are we going to release the last version?

**Comment by chrisgraham24** [comment]
3.8.4 did not fix this for me.

**Pull request merged**